src/main/java/org/csanchez/jenkins/plugins/kubernetes/pipeline/PodTemplateStep.java

```diff
@@ -161,6 +162,15 @@ public void setSlaveConnectTimeout(int slaveConnectTimeout) {
         this.slaveConnectTimeout = slaveConnectTimeout;
     }
 
+    public int getDeadlineSeconds() {
```

Comment:
Needs to be added to `PodTemplateStep/config.jelly` (and the corresponding help html field) so that the "pipeline syntax" generator includes this.

You can also use a number field for this, see `ContainerLogStep/config.jelly`.

Reply:
Changed to number field and added to ```PodTemplateStep/config.jelly``` so that field apperas in Pipeline syntax. Thanks for the tips!